### PR TITLE
Criteo - Update Publisher Tag version referenced by prebid adapter

### PR DESCRIPTION
**Pull request by leonardlabat**
## Type of change
- [X] Refactoring (no functional changes, no api changes)

## Description of change
Upgrade Criteo prebid adapter to reference the latest Criteo PubTag version.

### CODE REVIEW

**Comment by patmmccann** [comment]
Not yet available at the referenced repo

File: modules/criteoBidAdapter.js [hunk 1]
```diff
@@ -24,7 +24,7 @@ const LOG_PREFIX = 'Criteo: ';
   Unminified source code can be found in the privately shared repo: https://github.com/Prebid-org/prebid-js-external-js-criteo/blob/master/dist/prod.js
 */
 const FAST_BID_VERSION_PLACEHOLDER = '%FAST_BID_VERSION%';
-export const FAST_BID_VERSION_CURRENT = 117;
+export const FAST_BID_VERSION_CURRENT = 123;
```
Comment (patmmccann) [inline modules/criteoBidAdapter.js:27]: The repo in the line above stops at 116 currently

**Review by patmmccann — CHANGES_REQUESTED** [review]

File: modules/criteoBidAdapter.js (hunk 1 above)
Comment (leonardlabat) [reply modules/criteoBidAdapter.js:27]: Indeed it looks like the updates were not uploaded there
We've fixed it, thanks for reporting it Patrick

**Comment by patmmccann** [comment]
I read the pubtag js just now

A couple question:

Could you disclose the identity functionality in https://github.com/prebid/prebid.github.io/blob/master/dev-docs/bidders/criteo.md ? It seems like most of the functionalities are well disclosed in prebid issues but not in the primary documentation page. 

In that same file it mentions pubtag is required for floors. It seems like an oversight that hasn't been moved to your adapter yet. Could you "promote" that functionality into the main adapter soon?

**Comment by patmmccann** [comment]
When you add the link to the docs pull, happy to merge

**Pull request merged**